lxd/container_lxc.go

```diff
@@ -1852,10 +1836,7 @@ func (c *containerLXC) OnStop(target string) error {
 		deviceTaskSchedulerTrigger("container", c.name, "stopped")
 
 		// Record current state
-		err = dbContainerSetState(c.daemon.db, c.id, "STOPPED")
-		if err != nil {
-			return
-		}
+		dbContainerSetState(c.daemon.db, c.id, "STOPPED")
```

Comment:
Logging would probably be fine, returning on error was actually a bug as the error wouldn't be reported anywhere and if the container was ephemeral, we'd fail to delete it.